packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -428,15 +428,19 @@ export class StripeWebhookHandler extends StripeHandler {
       }
 
       const uid = customer.metadata.userid;
-      const account = await this.db.account(uid);
-
+      const account = await Account.findByUid(uid, { include: ['emails'] });
```

Comment:
Why does this need the `{ include: ['emails'] }' option?

Reply:
It doesn't. I copied the call from the function below. I can remove it.